test/common.js

```diff
@@ -165,7 +165,7 @@ Object.defineProperty(exports, 'opensslCli', {get: function() {
 
   if (exports.isWindows) opensslCli += '.exe';
 
-  var openssl_cmd = child_process.spawnSync(opensslCli, ['version']);
+  const openssl_cmd = child_process.spawnSync(opensslCli, ['version']);
```

Comment:
Now that you are changing this line, can you change the variable name to use camelCase?